packages/sdk-react/src/components/counterparts/helpers.ts

```diff
@@ -11,8 +11,33 @@ export const isOrganizationCounterpart = (
 ): counterpart is components['schemas']['CounterpartOrganizationRootResponse'] =>
   counterpart.type === 'organization';
 
-export function getIndividualName(firstName: string, lastName: string): string {
-  return `${firstName} ${lastName}`;
+export function getIndividualName(name: {
```

Comment:
`getIndividualName` is exported three times

Reply:
Yep, this is required for [function overloads](https://www.typescriptlang.org/docs/handbook/2/functions.html#function-overloads)